lmms_eval/tasks/mmrefine/mmrefine_evals.py

```
@@ -0,0 +1,169 @@
+                "solution_correctness": solution_correctness,
+            }
+
+    def create_one_query(self, problem):
+        query = REFINEMENT_PROMPT.format(question=problem["question"], initial_solution=problem["initial_solution"])
+        return query
+
+    def eval_result(self, result, config):
+        # extract and score for each question
+        full_prediction = result["prediction"].strip()
+        problem = {
+            "answer": result["answer"] if "answer" in result else None,
```

Ruff (0.12.2)

Use `result.get("answer", None)` instead of an `if` block

Replace with `result.get("answer", None)`

(SIM401)